en/controllers/middleware.rst

```diff
@@ -229,7 +225,9 @@ their own response. We can see both options in our simple middleware::
                     'expire' => '+ 1 year',
                 ]);
             }
-            return $response;
+            
+            // Pass the modified instances to the next Middleware
+            return $next($request, $response);
```

Comment:
Those middlewares all either handle the request creating a response or modify the request and not the response.

The code in these changes would result in responses without a cookie if any lower layer/controller *replaced* the response with a new one.